examples/sha256/main.rs

```diff
@@ -362,8 +362,9 @@ fn get_or_create_commitment_key<C: CurveAffine>(
 }
 
 fn main() {
-    env_logger::init();
-    log::info!("Start");
+    tracing_subscriber::fmt::init();
```

Comment:
Do we always init the tracing subscriber before using it? If that's the case, the use of "tracing::info/tracing::debug" in "src/lib" will only work when we call it from examples?

Reply:
Yes, this applies to all logs fraemworks in Rust. This is done to for example make it possible to write to the log collector system directly instead of to stdout

Reply:
Like if you have binary target - you should call any logs runner (here is tracing)